engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceModificationProcessor.java

```diff
@@ -46,6 +48,10 @@
       "Expected to modify process instance but no process instance found with key '%d'";
   private static final String ERROR_MESSAGE_TARGET_ELEMENT_NOT_FOUND =
       "Expected to activate element but no element found in process '%s' for element id(s): '%s'";
+  private static final String ERROR_MESSAGE_TARGET_ELEMENT_IN_MULTI_INSTANCE_BODY =
+      "Expected to modify instance of process '%s' with activate instructions but the element(s) "
+          + "with id(s) '%s' is inside a multi-instance subprocess. The activation of element(s) "
+          + "inside a multi-instance subprocess is not supported.";
```

Comment:
> with id(s) '%s' **is** inside

The usage of `is` is only correct when there is one id, but if there are multiple this should be changed to `are`. 

🔧 Perhaps you can find a different form of the same sentence that doesn't care about this plurality so much. This could also make the sentence easier to read, i.e. there would be fewer usages of `id(s)` and `element(s)`.

👍 I really like that the rejection message states what is not supported. 

This also highlights to me that the error messages are not aligned.

🔧 Please align the error messages.